src/Microsoft.AspNetCore.Mvc.TagHelpers/FormTagHelper.cs

```diff
@@ -173,9 +173,29 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                             RouteValuesPrefix));
                 }
 
-                // User is using the FormTagHelper like a normal <form> tag. Antiforgery default should be false to
-                // not force the antiforgery token on the user.
-                antiforgeryDefault = false;
+                string attributeValue = null;
+                switch (actionAttribute.Value)
+                {
+                    case HtmlString htmlString:
+                        attributeValue = htmlString.ToString();
+                        break;
+                    case string stringValue:
+                        attributeValue = stringValue;
+                        break;
+                }
+
+                if (string.IsNullOrEmpty(attributeValue))
+                {
+                    // User is using the FormTagHelper like a normal <form> tag that has an empty action attribute.
+                    // i.e. <form action="" method="post">
+                    antiforgeryDefault = true;
+                }
+                else
+                {
+                    // User is using the FormTagHelper like a normal <form> tag. Antiforgery default should be false to
+                    // not force the antiforgery token on the user.
+                    antiforgeryDefault = false;
```

Comment:
If we leave this alone (pending Monday's discussion), update the comments to better distinguish things from the `if` case e.g. `User is likely using the <form> element to submit to another site. Do not send an antiforgery token to unknown sites.`